website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,191 @@
+- [Pod Labels](#pod-labels) (e.g., GPU)
+
+## Instance Type Allowlist
+
+Karpenter supports specifying [AWS instance type](https://aws.amazon.com/ec2/instance-types/). 
+
+If one instance type is listed, Karpenter will always provision that type.
+
+If more than one type is listed, Karpenter will intelligently select the
+instance type to maximize node utilization.
+
+The default value includes all instance types with the exclusion of metal
+(non-virtualized),
+[non-HVM](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/virtualization_types.html),
+and GPU instances. View the full list with `aws ec2 describe-instance-types`.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  instanceTypes:
+    - m5.large
+```
+
+*Override with workload manifest (e.g., pod)*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        node.k8s.aws/instance-type: m5.large
+```
+
+## Node Lables in Provisioner Spec
+
+### Launch Template
+
+Karpenter uses [AWS Bottlerocket OS](https://aws.amazon.com/bottlerocket/) by
+default. More specifically, Karpenter automatically creates a launch template
+with the name `Karpenter-<cluster name>-uuid` for each region where a node is
+provisioned.
+
+You can specify a different [launch
+template](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html),
+such as a customized version of Bottlerocket or a different platform
+altogether.
+
+**Template ID**
+- key: `node.k8s.aws/launch-template-id`
+- value example: `bottlerocket` (default)
+- value list: `aws ec2 describe-launch-templates`
+
+### Capacity Type (e.g., spot)
+
+- key: `node.k8s.aws/capacity-type`
+- values
+  - `on-demand` (default)
+  - `spot`
+
+Karpenter supports specifying capacity type and defaults to on-demand.
+
+Specify this value on the provisioner to enable spot instances. [Spot
+instances](https://aws.amazon.com/ec2/spot/) may be preempted, and should not
+be used for critical workloads.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  labels: 
+    node.k8s.aws/capacity-type: spot
+```
+
+*Override with workload manifest (e.g., pod)*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        node.k8s.aws/capacity-type: spot
+```
+
+### Architecture (e.g., ARM) 
+
+- key: `kubernetes.io/arch`
+- values
+  - `amd64` (default)
+  - `arm64`
+
+Karpenter supports `amd64` nodes, and `arm64` nodes. 
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  labels: 
+    kubernetes.io/arch: arm64
+```
+
+*Override with workload manifest (e.g., pod)*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+```
+
+## Operating System
+
+- key: `kubernetes.io/os`
+- values
+  - `linux` (default)
+
+At this time, Karpenter only supports Linux OS nodes.
+
+### Availability Zones
+
+`topology.kubernetes.io/zone=us-east-1c`
+
+- key: `topology.kubernetes.io/zone`
+- value example: `us-east-1c` or `use1-az1`

```

Comment:
I'm not sure that az ids work. I think there was a technical blocker.

Reply:
according to @bwagner5 they do?

Reply:
They definitely did at one point, unless something regressed. I can check tomorrow, but they need to be supported, so if it has regressed I’ll open an issue to fix.

Reply:
This has been reverted.